osdteletext.c

```diff
@@ -340,6 +346,24 @@ cTeletextSetupPage::cTeletextSetupPage(void) {
    cString buf;
    cOsdItem *item;
 
+temp.txtBlock[0] =  tr("Latin 1");
+temp.txtBlock[1] =  tr("Latin 2");
+temp.txtBlock[2] =  tr("Latin 3");
+temp.txtBlock[3] =  tr("Latin 4");
+temp.txtBlock[4] =  tr("Cyrillic");
+temp.txtBlock[5] =  tr("Reserved");
+temp.txtBlock[6] =  tr("Greek");
+temp.txtBlock[7] =  tr("Reserved");
+temp.txtBlock[8] =  tr("Arabic");
+temp.txtBlock[9] =  tr("Reserved");
+temp.txtBlock[10] = tr("Hebrew");
+temp.txtBlock[11] = tr("Reserved");
+temp.txtBlock[12] = tr("Reserved");
+temp.txtBlock[13] = tr("Reserved");
+temp.txtBlock[14] = tr("Reserved");
+temp.txtBlock[15] = tr("Reserved");
```

Comment:
The indentatation is off. Define this as a class member / local variable instead of part of the TeletextSetup and remove one additional linefeed added.